src/shared/modules/auth/helpers.js

```diff
@@ -3,15 +3,17 @@ import { AUTH_LOGGING_PREFIX, AUTH_STORAGE_LOGS } from './constants'
 import { isAuthLoggingEnabled, isAuthDebuggingEnabled } from './settings'
 import { saveAs } from 'file-saver'
 
+const MAX_LOG_LINES = 200
 export const authLog = (msg, type = 'log') => {
   if (!isAuthLoggingEnabled) return
   if (!['log', 'error', 'warn'].includes(type)) return
-  const log = `${AUTH_LOGGING_PREFIX} [${new Date().toISOString()}] ${msg}`
+  const messageNoNewlines = msg.replace('\n', ' ')
```

Comment:
hmm, I don't have a better idea, but replacing new line with space seems like it might make messages hard to read...

Reply:
No messages should have a newline, this is just a failsafe to make sure the logs are still easily parse-able